packages/dd-trace/src/remote_config/index.js

```diff
@@ -0,0 +1,260 @@
+'use strict'
+
+const uuid = require('crypto-randomuuid')
+const { EventEmitter } = require('events')
+const Scheduler = require('./scheduler')
+const tracerVersion = require('../../../../package.json').version
+const request = require('../exporters/common/request')
+const log = require('../log')
+
+const clientId = uuid()
+
+const POLL_INTERVAL = 5e3
+
+// There MUST NOT exist separate instances of RC clients in a tracer making separate ClientGetConfigsRequest
+// with their own separated Client.ClientState.
+class RemoteConfigManager extends EventEmitter {
+  constructor (config) {
+    super()
+
+    this.scheduler = new Scheduler((cb) => this.poll(cb), POLL_INTERVAL)
```

Comment:
We no longer use a scheduler for tracing and instead just schedule as needed after the task has run once. It would be more consistent to do this here, and it would also make it easier to migrate to long-polling later on.

Definitely not a blocker and it's fine either way.

Reply:
I don't think I understand how that could work here. Traces are pushing info to the backend, here we are polling info from the backend, we don't have a queue of tasks, the backend does.

Reply:
I guess the timeout is smaller than the poll interval anyway, so it doesn't really matter and can be addressed later if needed.